### ISSUE TITLE
Polymer 2.x/Polyfills produce incorrect scopeSubtree result when switching from 1.x to 2.x

### ISSUE THREAD

**Issue by ergo** [Polymer/polymer#4809]
Test element showing differences in behavior between 1.x and 2.x polymer/polyfills.

I've prepared bower_1_9.json and bower_2_0.json to quickly switch between polymer versions.

To test just visit the demo page under /components/polymer-quilljs/demo/.

When using polymer 1.9 the editor renders correctly under firefox 55, 
when using polymer 2.x with latest polyfills the rendering of editors toolbar is broken. 

#### Live Demo
https://github.com/ergo/polymer-quilljs-css-test
Minimal demo - sorry no easy way to use qilljs in jsbin I think

#### Steps to Reproduce
1. run bower install 
2. visit the demo in firefox

#### Expected Results
The editor toolbar is rendered correctly
When running the same demo with depenencies from bower_1_9.json everything is rendered correctly .

#### Actual Results
Some elements are overflowing and don't render correctly at all

### Browsers Affected
- [x ] Firefox

**Comment by sorvell** [comment]
In Polymer 2.x `scopeSubtree` is a no-op. The Shady CSS polyfill is supposed to handle this behavior now and there's likely an issue there. @azakus mind taking a look?

**Comment by ergo** [comment]
@sorvell OK, so there is some difference in how the polyfill is handling this. Thanks for the update.

**Comment by sorvell** [comment]
Looks like there's more than one issue here. 

1. The style scoping is broken because of a broken optimization in ShadyCSS. Looks like we can address this without nerfing the optimization too much.

1. Looks like there's another issue with the events that we're also tracking down.

**Comment by ergo** [comment]
Look like those are fixed. Any chance we can get a new release with the fixes soon?

**Comment by TimvdLippe** [comment]
I think the plan was to collect some more bugfixes and then also publish a new Polymer version, @azakus ?

**Comment by ergo** [comment]
As someone who got blocked pretty hard by issues I reported recently I would love to see more releases from Polymer and webcomponents polyfills. It is a library so frequent bugfix releases are normal. Right now I'm hard blocked on 1.x to 2.x migration because application does not render correctly for me at all on firefox. I'm not sure how much overhead this add on project leader, but I think frequent releases would be great.

I want to start a series of tutorial on polymer but I really can't release something and say "this only works on chrome right now until it's fixed".

**Comment by sorvell** [comment]
Closing this since both the [noted issues](https://github.com/Polymer/polymer/issues/4809#issuecomment-326695300) have been addressed.
